massive-data-import/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+
+services:
+  kuzzle:
+    image: kuzzleio/howto-massiveimport-kuzzle
+    volumes:
+      - "./scripts:/scripts"
+    depends_on:
+      - redis
+      - elasticsearch
+    ports:
+      - "7512:7512"
+    environment:
+      - kuzzle_services__db__client__host=http://elasticsearch:9200
+      - kuzzle_services__internalCache__node__host=redis
+      - kuzzle_services__memoryStorage__node__host=redis
+      - kuzzle_server__maxRequestSize=1GB
```

Comment:
For this parameter and the one just below, I think these would be far easier to document by putting them in a `/etc/kuzzlerc` file and mentioning that configuration in the how-to (see my comments above).